### DISCUSSION TITLE
Feature Request: excalidraw extension

### DISCUSSION THREAD

**Discussion by waveiron** [ueberdosis/tiptap#3959] [Feature Requests]
### Description

Official extension for free hand drawing tool like these:
https://lexical.dev/docs/demos/plugins/excalidraw
https://plate.udecode.io/docs/plugins/excalidraw

### Use Case

<img width="1153" alt="image" src="https://user-images.githubusercontent.com/4930447/232176476-f31296df-a9ba-4492-8290-4c61a85a8b42.png">

### Type

New extension

**Comment by Rieranthony** [comment]
I would LOVE this one! ☝️
